docs/content/docs.md

```diff
@@ -552,6 +552,15 @@ This flag can be useful for debugging and in exceptional circumstances
 (eg Google Drive limiting the total volume of Server Side Copies to
 100GB/day).
 
+### --dscp VALUE ###
+
+Specify a DSCP value or name to use in connections. This could help QoS
+system to identify traffic class. EF, DF, CSx and AFxx are allowed.
+
```

Comment:
This should mention that this is only effective for HTTP based services as your patch doesn't affect the ftp/sftp backends.

Reply:
Should I also implement for ftp backends?

Reply:
Nice one :-)

You appear to have a conflict though so you'll have to rebase on master.

Do you want to do backend/sftp too?

Reply:
I thought sftp already uses fshttp.NewDialer?

Reply:
So it does! I'd forgotten that :-)

Reply:
So this thread is resolved.